BlueBackPixel.java

```diff
@@ -579,33 +569,25 @@ private void openClampLittle(string wait) {
   //**********************************************
   // double functin same idea as above
   private void reverseFlipper() { flipper.setPosition(CLAW_FLIP_SERVO_FLIPPED_POS); }
-  private void reverseFlipper(string wait) {
-    flipper.setPosition(CLAW_FLIP_SERVO_FLIPPED_POS);
-    if (wait == "wait") {
-      until (flipper.getPosition == CLAW_FLIP_SERVO_FLIPPED_POS) { 
-        sleep(1);
-      }
-    }
-    sleep(5);
+  private void reverseFlipperWait() {
+    double distance = Math.abs(grabber.getPosition - CLAW_FLIP_SERVO_FLIPPED_POS);
+    grabber.setPosition(CLAW_FLIP_SERVO_FLIPPED_POS);
+    sleep(distance * CALC_FLIPPER_WAIT_MS + 5);
```

Comment:
@gwarby I'm not sure if you caught & fixed this during Tuesday's meeting, but this Flipper function has grabber calls in it.